test/spec/oidc/getUserInfo.ts

```diff
@@ -134,7 +134,7 @@ describe('token.getUserInfo', function() {
       url: accessToken.userinfoUrl,
       method: 'GET',
       headers: {
-        DPoP: 'dpopproofyay',
+        Dpop: 'dpopproofyay',
```

Comment:
Isn't the header supposed to be `DPoP`?  https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-16#name-the-dpop-http-header

Reply:
> Note that per [[RFC9110](https://www.rfc-editor.org/info/rfc9110)] header field names are case-insensitive; so DPoP, DPOP, dpop, etc., are all valid and equivalent header field names. Case is significant in the header field value, however.
-https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-16#section-4.1-5